src/main/java/io/quarkus/bot/PushToProjects.java

```diff
@@ -72,12 +72,17 @@ void pullRequestLabeled(@PullRequest.Labeled GHEventPayload.PullRequest pullRequ
             return;
         }
 
```

Comment:
Thanks! Obviously, I did all my tests in an organization.

Please make it a condition here with a `return` so that the flow stays simple e.g.

```
if (pullRequestPayload.getOrganization() == null) {
    return;
}
```
I do that for all edge cases.

Also:
- no need to log something: it's just gonna pollute the log for no good reason
- we need to have the same thing for the issues above